src/commands/generate/typeGenerator.ts

```diff
@@ -115,21 +119,32 @@ export async function generateEnum(definition: IEnumDefinition, simpleAst: Simpl
     const sourceFile = simpleAst.createSourceFile(definition.typeName);
 
     if (definition.values.length > 0) {
-        const namespaceDefinition = sourceFile.addNamespace({
+        const typeAliases = definition.values.map<TypeAliasDeclarationStructure>(enumValue => ({
+            kind: StructureKind.TypeAlias,
+            isExported: true,
+            name: enumValue.value,
+            type: doubleQuote(enumValue.value),
+            docs: addDeprecatedToDocs(enumValue),
+        }));
+        typeAliases[typeAliases.length - 1].trailingTrivia = `\n\n`;
+        const variableDeclarations = definition.values.map<VariableStatementStructure>(enumValue => ({
+            kind: StructureKind.VariableStatement,
+            isExported: true,
+            declarationKind: VariableDeclarationKind.Const,
+            declarations: [
+                {
+                    name: enumValue.value,
+                    initializer: `${doubleQuote(enumValue.value)} as ${doubleQuote(enumValue.value)}`,
+                },
+            ],
+        }));
+
+        const namespaceDefinition = sourceFile.addModule({
+            kind: StructureKind.Module,
+            declarationKind: ModuleDeclarationKind.Namespace,
```

Comment:
@gluxon ts-morph is weird in the way it handles namespaces vs modules. But this is what makes it a namespace.